setup.cfg

```diff
@@ -78,7 +78,15 @@ testpaths = "jdaviz" "docs"
 astropy_header = true
 doctest_plus = enabled
 text_file_format = rst
-addopts = --doctest-rst --import-mode=append -Wignore
+addopts = --doctest-rst --import-mode=append
+filterwarnings =
+    error
+    ignore:Numpy has detected that you:DeprecationWarning
```

Comment:
Something about striding tricks and overlapping memory... This is the biggest offender. I have a feeling this is a rabbit hole straight into Glue. A battle for another day.